tsconfig.json

```diff
@@ -1,14 +1,17 @@
 {
-	"compilerOptions": {
```

Comment:
Changed tabs to 2 spaces (to match package.json whose indent is automanaged by npm), and added the last two compiler options